pages/api/hunks.ts

```diff
@@ -0,0 +1,14 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { saveHunk } from '../../utils/db/relevance';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
The convention I have followed in this whole repository is using the arrow function notation. I'd like it if we keep following that.

instead of `function fn_name() {}`, use the `const fn_name = () => {}` syntax.